src/ws-tiles-chart/ws-tiles-chart.js

```diff
@@ -24,7 +29,10 @@ export class WSTilesChart extends Component {
     maxTileSize: 25,
     minTileSize: 8,
     width: 80,
-    height: 80
+    height: 80,
```

Comment:
just a question, because I was not involved in the Design talks, are these default values from UX or did you choose some on your own ?